policy/configurations/comcam.yaml

```diff
@@ -0,0 +1,12 @@
+description: LSST ComCam Parameters
+instrument: lsst.obs.lsst.ComCam
+
+# Pupil inner and outer radius
+pupil:
+  R_inner: 0.0
+  R_outer: 0.335
```

Comment:
Please add the unit to the radius. This applies to all the `yaml` files. For any physical amounts, you would need to add the units in the comments. In addition, please rename them to be `radiusInner` and `radiusOuter` in `pupil` and `obscuration` fields.